sdk-extensions/autoconfigure/README.md

```diff
@@ -82,10 +82,10 @@ The [OpenTelemetry Protocol (OTLP)](https://github.com/open-telemetry/openteleme
-| otel.exporter.otlp.compression                           | OTEL_EXPORTER_OTLP_COMPRESSION                           | The compression type to use on OTLP trace, metric, and log requests. Options include `gzip`. By default no compression will be used.                                                                                                                                                                                                                                                                                 |
-| otel.exporter.otlp.traces.compression                    | OTEL_EXPORTER_OTLP_TRACES_COMPRESSION                    | The compression type to use on OTLP trace requests. Options include `gzip`. By default no compression will be used.                                                                                                                                                                                                                                                                                                  |
-| otel.exporter.otlp.metrics.compression                   | OTEL_EXPORTER_OTLP_METRICS_COMPRESSION                   | The compression type to use on OTLP metric requests. Options include `gzip`. By default no compression will be used.                                                                                                                                                                                                                                                                                                 |
-| otel.exporter.otlp.logs.compression                      | OTEL_EXPORTER_OTLP_LOGS_COMPRESSION                      | The compression type to use on OTLP log requests. Options include `gzip`. By default no compression will be used.                                                                                                                                                                                                                                                                                                    |
+| otel.exporter.otlp.compression                           | OTEL_EXPORTER_OTLP_COMPRESSION                           | The compression type to use on OTLP trace, metric, and log requests. Options include `none` and `gzip`. By default no compression will be used.                                                                                                                                                                                                                                                                                 |
```

Comment:
If we're going to add none as an option here, should update the default phrasing to make it clear that none is the default. I.e. "By default no compression will be used" => "Default is `none`."